<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dietetique</class>
 <widget class="QMainWindow" name="dietetique">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dietetique</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>421</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Informations personnelles</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>261</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Nom">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditNom"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Prenom">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Prénom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPrenom"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Sexe">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Sexe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>Femme</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>Homme</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Age">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Age</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxAge"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Poids">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Poids (kg)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBoxPoids"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Taille">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Taille (m)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Taille"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Suite</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Calcul du poids idéal</string>
     </attribute>
     <widget class="QTextEdit" name="textEditAfficheur">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>391</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>307</width>
        <height>86</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Poids idéal formule de devine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Poids idéal formule de lorentz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Poids idéal formule de lorentz en fonction de l'age</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
